dogfood/coder/files/etc/apt/preferences.d/docker

```diff
@@ -4,8 +4,12 @@ Pin: origin download.docker.com
 Pin-Priority: 1
 
 # Docker Community Edition
+# We need to pin docker-ce to a specific version because containerd is pinned
+# to an older version. Newer major versions of docker-ce require a version of
+# containerd.io greater than our pinned version.
 Package: docker-ce
 Pin: origin download.docker.com
+Pin: version 5:27.*
```

Comment:
Docker 28 stable was released on October 8, which seems to roughly check out with when the dogfood action started failing https://docs.docker.com/engine/release-notes/28/